tests/ui/layout/unsafe-cell-hides-niche.rs

```diff
@@ -55,14 +55,14 @@ macro_rules! check_sizes {
 const PTR_SIZE: usize = std::mem::size_of::<*const ()>();
 
 check_sizes!(Wrapper<u32>:     4 => 8);
-check_sizes!(Wrapper<N32>:     4 => 4); // (✓ niche opt)
+check_sizes!(Wrapper<NonZero<u32>>:     4 => 4); // (✓ niche opt)
```

Comment:
If you are planning to push other changes anyway:

I think adding `type N32 = std::num::NonZero<u32>;` above this block would allow it to preserve the tabular alignment intended by the original author.